README.md

```diff
@@ -33,11 +35,41 @@ julia> mortar(reshape([randn(2,2), randn(1,2), randn(2,3), randn(1,3)],2,2))
   0.430093  -0.0263753  │  -1.31275     0.278447  -0.139579
 ```
 
+Alternatively, one can add block structure on top of an existing array by wrapping the array in  `BlockedArray`, where the extra arguments give the sizes of the blocks:
+```julia
+julia> BlockedArray(randn(7), [3,4])
+2-blocked 7-element BlockedVector{Float64}:
+ -0.17348560551451797
+ -0.5680124317024628
+  1.699007590285868
+ ─────────────────────
+ -0.7437814954416642
+ -0.018198226033108045
+  1.3335354818213445
+ -0.03512135185007728
+
+julia> BlockedArray(randn(3,5), [2,1], [2,3])
+2×2-blocked 3×5 BlockedMatrix{Float64}:
+  0.444186   0.788823  │   0.743428  -0.815026   0.715779
+ -0.721074  -0.43783   │   1.07413   -0.336926   0.539873
+ ──────────────────────┼─────────────────────────────────
+  0.128836  -0.350202  │  -2.71365    1.67605   -0.25611
+```
+
+
 ## Documentation
 
 - [**STABLE**][docs-stable-url] &mdash; **most recently tagged version of the documentation.**
 - [**LATEST**][docs-dev-url] &mdash; *in-development version of the documentation.*
 
+## Changes in v1.0
+
+We are excited to release v1.0! There are some important breaking changes from previous versions of BlockArrays.jl:
+
+- `BlockedArray` replaces `PseudoBlockArray`.
+- Axes are now typically `BlockedOneTo` instead of `BlockUnitRange`.
+- Support for some simple block-banded matrices has been moved here from BlockBandedMatrices.jl
```

Comment:
Other things to potentially mention:
- The definition of `blocksizes(array::AbstractArray)` is changed from `blocklengths.(axes(array))` to an iterator of size `blocksize(array)` over the sizes of each block of `array`.
- `BlockedUnitRange` is now parametrized by the element type instead of hardcoded to `Int`.